testing/userclk/test_userclk_c.cpp

```diff
@@ -322,7 +330,196 @@ TEST_P(userclk_c_p, main2) {
  *             then low is calculated from high,<br>
  *             and the fn returns zero.<br>
  */
-TEST_P(userclk_c_p, main3) {
+TEST_P(userclk_c_hw_p, main3) {
+  char zero[20];
+  char one[20];
+  char two[20];
+  char three[20];
+  char four[20];
+  char five[20];
+  char six[20];
+  char seven[20];
+  char eight[20];
+  char nine[20];
+  char ten[20];
+  char eleven[20];
+  char twelve[20];
+  strcpy(zero, "userclk");
+  strcpy(one, "--segment");
+  sprintf(two, "%d", platform_.devices[0].segment);
+  strcpy(three, "-B");
+  sprintf(four, "%d", platform_.devices[0].bus);
+  strcpy(five, "-D");
+  sprintf(six, "%d", platform_.devices[0].device);
+  strcpy(seven, "-F");
+  sprintf(eight, "%d", platform_.devices[0].function);
+  strcpy(nine, "-S");
+  if (platform_.devices[0].bus == 129) { strcpy(ten, "0x0"); }
```

Comment:
Did you touch base with @r-rojo on this one?  Is there a better way we can do this?